### ISSUE TITLE
Pressing tab selects "More" before the search box

### ISSUE THREAD

**Issue by mstriemer** [mozilla/addons-frontend#4021]
### Describe the problem and steps to reproduce it:
1. Load AMO
2. Press tab

### What happened?
The "More" dropdown is selected

### What did you expect to happen?
The search box is focused

### Anything else we should know?
Adding `tabindex="0"` to the search box makes it get focused first but then tabbing continues down the page instead of moving from the search box/submit button to the "More" menu. I'm not sure what the correct order would be though.

Marking this accessibility since there could be some considerations there.

**Comment by muffinresearch** [comment]
I get the first link in the page selected (which is the logo link) not the "More" dropdown. Do you have something setup causing tab to not focus links by default?

**Comment by mstriemer** [comment]
I thought the default was just to select inputs, buttons, selects, etc. I think it's targeting the "More" link for me because it's a button, not a link.

Interestingly Safari doesn't select buttons with tab unless you're in VoiceOver mode.

It looks like if you set `tabindex="1"` (or anything greater than 0, it seems) then it will first focus that then move back to the top of the page and tab through. That might be the best solution here.

**Comment by muffinresearch** [comment]
Starting Firefox with a new profile I cannot reproduce this. 

@mstriemer I'm going to close but please can you try with a new profile - feel free to re-open if you can still repro this with a new profile.
